src/main/java/com/katelocate/menugenerator/recipe/RecipeController.java

```diff
@@ -31,6 +33,23 @@ Recipe findById(@PathVariable Integer id) {
         return recipe.get();
     }
 
+    @GetMapping("/day")
+    ArrayList<Recipe> getDayMenu() {
+        RecipeType[] types = {
+                RecipeType.BREAKFAST,
+                RecipeType.SNACK,
+                RecipeType.DINNER,
+        };
+        Random random = new Random();
+        ArrayList<Recipe> dayMenu = new ArrayList<>(3);
+        for (RecipeType type : types) {
+            List<Recipe> choices = recipeRepository.findByType(type);
+            int randomIndex = random.nextInt(choices.size());
+            dayMenu.add(choices.get(randomIndex));
+        }
+        return dayMenu;
```

Comment:
Let's add tests for this method as well.
I see at least two:
- Check that it returns expected menu, if you have only one breakfast, snack and dinner
- Check that it returns both menus, if you have e.g. 2 breakfasts, 1 snack and 1 dinner